meson.build

```diff
@@ -181,4 +181,22 @@ top_earlgrey = declare_dependency(
   )
 )
 
+# ROM_EXT manifest generator.
+rom_exts_source_dir = meson.source_root() / 'sw/device/rom_exts'
+custom_target(
+  'rom_exts_manifest_h',
+  output: 'manifest.h',
+  depend_files: [
+    rom_exts_source_dir / 'manifest.h.tpl',
+    rom_exts_source_dir / 'manifest.hjson',
+  ],
+  command: [
+    prog_python,
+    meson.source_root() / 'util/rom-ext-manifest-generator.py',
+    '--input-dir', rom_exts_source_dir,
+    '--output-dir', rom_exts_source_dir,
```

Comment:
This should be generated into `meson.build_root()`, but in the equivalent directory - you shouldn't need to update the inclusion path where this is used, as `meson.build_root()` is added to the compiler include path.

Reply:
Ok, I actually realised there were a few nonstandard ways you were using Meson here. I've prepared a patch that we can talk through on Monday that improves this - it's here: https://github.com/lenary/opentitan/commit/396554cb718525ef00ffc3d214c21cf7419a89f7

Reply:
The idea was that this would be checked-in file. I thought it would be useful to make it a checked in file, hence it is generate d into the source directory.